tests/rptest/tests/cloud_storage_scrubber_test.py

```diff
@@ -621,6 +621,9 @@ def test_scrubber(self, cloud_storage_type):
         self._produce()
         self._assert_no_anomalies()
 
+        self.redpanda.set_cluster_config(
```

Comment:
if scrubbing is disabled, what is the purpose of the above `self._assert_no_anomalies()`? shall it be moved after cluster config is changed?

Reply:
it checks that we don't start with some anomalies I suppose

Reply:
But you just disabled scrubbing at start, why we still have the assert in place? It makes no sense to me

Reply:
i reverted this change